samples/tfm_integration/psa_level_1/docs/attestation.md

```diff
@@ -0,0 +1,274 @@
+# Initial Attestation
```

Comment:
Hmm.. Why isn't this a .rst file instead of a markdown file?  As is, it won't be included in our generated documentation. Check out ``pandoc`` as a quick way to convert ``.md`` to ``.rst``

Reply:
@microbuilder

Reply:
I can convert this to rst if you’d like.

Reply:
https://github.com/zephyrproject-rtos/zephyr/commit/0d3d2d85915889fffd9a077cbc8e5809a13a07c2
This commit can be amended afterward I think.